commons/commons-lang/src/test/java/com/clicktravel/common/random/RandomsTest.java

```diff
@@ -292,4 +334,14 @@ public void shouldReturnRandomId() {
         assertEquals("Random sample ids should be unique", SAMPLE_SIZE, randomIds.size());
     }
 
+    @Test
+    public void shouldReturnRandomCreditCardNumber() {
```

Comment:
Should have another test to verify the check digit is correct. I suggest testing with examples and verify the check digit is as expected.